src/hammr/commands/template/template.py

```diff
@@ -293,11 +294,11 @@ def do_create(self, args):
                 #delete tmp dir
                 shutil.rmtree(constants.TMP_WORKING_DIR)
             os.mkdir(constants.TMP_WORKING_DIR)
-            file = open(constants.TMP_WORKING_DIR + os.sep + constants.TEMPLATE_JSON_NEW_FILE_NAME, "w")
```

Comment:
- We need to handle both YAML and JSON
- We need to create a dedicated function for this